src/account/session.ts

```diff
@@ -139,3 +139,16 @@ export class SessionAccount extends Account implements AccountInterface {
     );
   }
 }
+
+export async function supportsSession(
```

Comment:
Is this function maybe more suitable to be outside of this file, in some util (I don't see it depends on anything except SESSION_PLUGIN_CLASS_HASH from this file)? ...Because when looking at the rest of the projects code, I didn't see similar cases, just private functions outside classes

Reply:
Yes, it should be in a different file